apps/oxlint/src/output_formatter/mod.rs

```diff
@@ -40,6 +43,7 @@ impl FromStr for OutputFormat {
             "checkstyle" => Ok(Self::Checkstyle),
             "github" => Ok(Self::Github),
             "stylish" => Ok(Self::Stylish),
+            "junit" => Ok(Self::JUnit),
```

Comment:
in this file there is a snapshot test for the different formats. Could you please add one test for it too?
The test should also work in windows.

Reply:
Done